lib/fastlane/plugin/wpmreleasetoolkit/actions/common/gp_downloadmetadata_action.rb

```diff
@@ -68,6 +69,12 @@ def self.available_options
                                        env_name: 'FL_DOWNLOAD_METADATA_DOWNLOAD_PATH',
                                        description: 'The path of the target files',
                                        type: String),
+          FastlaneCore::ConfigItem.new(key: :auto_retry,
+                                       env_name: 'FL_DOWNLOAD_METADATA_AUTO_RETRY',
+                                       description: 'Whether to auto retry downloads after Too Many Requests error',
+                                       type: FastlaneCore::Boolean,
+                                       optional: true,
+                                       default_value: false),
```

Comment:
tbh I wouldn't mind having this set to `true`, given how useful it'll be to have this feature (even for repos that are still doing manual releases and not using release-on-CI) 🙃

Reply:
I wouldn't mind it either, but that'd be somewhat of a breaking change. If others feel the same way about setting it to `true`, it's easy enough to change :)

Reply:
+1 for setting to `true` for the convenience it would provide.

I can see how it would change the default behavior and be seen as a breaking change. However, given we are the only consumers, I think our threshold for when to introduce breaking changes is relatively low. I'd also be fine with this being a minor version bump, as updating to this version would not "break" anything.

Reply:
Addressed in dc56ef0ca29e7977db8b29b358f9d352d052b00f.